### ISSUE TITLE
Default sort order on homepage

### ISSUE THREAD

**Issue by TruffleClock** [Mirascope/lilypad#73]
### Description

I think the homepage should probably sort by reverse chron by default?

Not a big deal, since you can click the headers to sort, but I got confused for a bit when it seemed like it was autosorted for a while, then suddenly didn't see my new generation appear.

<img width="1048" alt="Screenshot 2024-12-18 at 11 03 47 PM" src="https://github.com/user-attachments/assets/3c559573-5c63-43cc-9ef0-3b996135618a" />

**Comment by willbakst** [comment]
Yeah we should definitely default the sorting to show the most recent trace at the top.
